continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -18,7 +18,7 @@ build:
 
 outputs:
   - name: distributed-impl
-    version: {{ version }}
+    version: {{ version }}+{{ GIT_DESCRIBE_HASH }}
```

Comment:
Should we add this to the other packages here as well?

Reply:
Ah nvm we have this in the `build/string` in other cases

Reply:
I'm fine with that! I think that the cython builds are young enough that slightly altering the build string shouldn't cause any issues 🙂